src/Sarif.Converters/ToolFormatConverter.cs

```diff
@@ -55,6 +56,8 @@ public ToolFormatConverter(AssemblyLoadFileDelegate assemblyLoadFileDelegate = n
 
             // FileMode settings here will results in an exception being raised if the input 
             // file does not exist, and that an existing output file will be overwritten
+            Stopwatch w = Stopwatch.StartNew();
```

Comment:
I will remove - debuggability while I was coding.

Reply:
Time to remove now? :)

---
In reply to: [283578926](https://github.com/microsoft/sarif-sdk/pull/1491#discussion_r283578926) [](ancestors = 283578926)

Reply:
Fixed.